backend/src/test/java/gov/cdc/usds/simplereport/api/TranslatorTest.java

```diff
@@ -34,40 +34,48 @@
 
 class TranslatorTest {
   @Test
-  void testEmptyShortDate() {
+  void emptyUserShortDate_returnsNull() {
     assertNull(parseUserShortDate(""));
   }
 
   @Test
-  void testNullShortDate() {
+  void nullUserShortDate_returnsNull() {
     assertNull(parseUserShortDate(null));
   }
 
   @Test
-  void testValidShortDate() {
+  void validUserShortDate_withStandardFormatParsesCorrectly() {
     LocalDate result = parseUserShortDate("2/1/2021");
     assertEquals(2, result.getMonthValue());
     assertEquals(1, result.getDayOfMonth());
     assertEquals(2021, result.getYear());
   }
 
   @Test
-  void testValidDateWithLeadingZeros() {
+  void validUserShortDate_withLeadingZerosParsesCorrectly() {
     LocalDate result = parseUserShortDate("02/01/2021");
     assertEquals(2, result.getMonthValue());
     assertEquals(1, result.getDayOfMonth());
     assertEquals(2021, result.getYear());
   }
 
   @Test
-  void testInvalidShortDate() {
+  void validUserShortDate_withShortYearParsesCorrectly() {
+    LocalDate result = parseUserShortDate("2/1/80");
+    assertEquals(2, result.getMonthValue());
+    assertEquals(1, result.getDayOfMonth());
+    assertEquals(1980, result.getYear());
+  }
+
+  @Test
+  void invalidUserShortDate_throwsException() {
     IllegalGraphqlArgumentException caught =
         assertThrows(
             IllegalGraphqlArgumentException.class,
             () -> {
-              parseUserShortDate("fooexample.com");
+              parseUserShortDate("0/0/23");
```

Comment:
thank you this is a much better test example! (even if it functionally doesn't matter)